code/controllers/subsystems/job.dm

```diff
@@ -457,7 +457,7 @@
 
 	var/datum/spawnpoint/spawnpos = SSatlas.spawn_locations["Cryogenic Storage"]
 	if(spawnpos && istype(spawnpos))
-		to_chat(src, "<span class='warning'>You come to the sudden realization that you never left the Aurora at all! You were in cryo the whole time!</span>")
+		to_chat(src, "<span class='warning'>You come to the sudden realization that you never left the [current_map.station_name] at all! You were in cryo the whole time!</span>")
```

Comment:
cryo doesn't exist in the horizon, i think. been replaced with living quarters.

Reply:
cryo + living quarters exist

Reply:
doesn't really matter if it exists or not, it should be dynamic regardless